vendor/knative.dev/pkg/apis/istio/common/v1alpha1/string.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 The Knative Authors
+
```

Comment:
2019

Reply:
These are vendored files. Nothing to do here for us.

Reply:
Oh.  That's right.